lib/stsci/tools/stpyfits.py

```diff
@@ -22,9 +22,85 @@
 
 PY3K = sys.version_info[0] > 2
 ASTROPY_VER_GE20 = LooseVersion(astropy.__version__) >= LooseVersion('2.0')
+ASTROPY_VER_GE32 = LooseVersion(astropy.__version__) >= LooseVersion('3.2')
 
 STPYFITS_ENABLED = False  # Not threadsafe TODO: (should it be?)
 
+if ASTROPY_VER_GE32:
+    from astropy.io.fits.header import _DelayedHeader, Header
+
+    class _NoDelayedHeader(_DelayedHeader):
```

Comment:
There is still the problem of `KeyError` where keys are not found because the entire header is not loaded. How do we force a header to be not delayed?